### ISSUE TITLE
beep in pink noise 

### ISSUE THREAD

**Issue by samo-1** [cliambrown/EasyNoise#17]
There's a slight beep circa every 4 seconds in the pink noise

**Comment by Zhormos** [comment]
Yes, I can hear it too, and there are repeating patterns in other noise types as well. It’s annoying because once you notice it, it grabs your attention.

I think the app uses short looping audio samples as noise sources, so our ears can detect repeating subtle patterns. 

This issue might be eliminated by adding a mathematically generated random-noise source in addition to the existing noises. I'm interested in implementing this myself later, if I can.

**Comment by cliambrown** [comment]
@Zhormos I love the way this tool works — it allows you to generate a custom noise based on your own hearing levels at different frequencies: https://mynoise.net/NoiseMachines/greyNoiseGenerator.php

I also found this customizable grey noise generator written in python: https://github.com/philousoph/Noise-Generator
